docs/tutorials/03_minimum_eigen_optimizer.ipynb

```diff
@@ -293,6 +296,247 @@
     "print(qaoa_result)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Analysis of Samples\n",
+    "Objects of type `SolutionSample` (here denoted as *samples*) are a useful information source provided\n",
+    "by `OptimizationResult` objects. They contain information about input values `x`, corresponding (optimized) function\n",
+    "values `fval`, probability of obtaining that result `probability` and the solution status `status` (`SUCCESS`,\n",
+    "`FAILURE`, `INFEASIBLE`)."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "variable order: ['x', 'y', 'z']\n",
+      "SolutionSample(x=array([0., 1., 0.]), fval=-2.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([0., 0., 0.]), fval=0.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([1., 1., 0.]), fval=0.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([1., 0., 0.]), fval=1.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([0., 0., 1.]), fval=3.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([1., 0., 1.]), fval=3.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([0., 1., 1.]), fval=3.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n",
+      "SolutionSample(x=array([1., 1., 1.]), fval=4.0, probability=0.12499999999999994, status=<OptimizationResultStatus.SUCCESS: 0>)\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('variable order:', [var.name for var in qaoa_result.variables])\n",
+    "for s in qaoa_result.samples:\n",
+    "    print(s)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We may also want to filter samples according to their status or to a certain percentile, i.e. with probability higher\n",
```

Comment:
according to their status or to a certain percentile, i.e., ... -> according to their status or probabilities. (omit i.e. part)